src/Mod/Path/Gui/Resources/icons/Path-TurnFace.svg

```diff
@@ -0,0 +1,1024 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
```

Comment:
This was created with Inkscape, and the file was saved as "Inkscape SVG". This introduces non-standard tags so it is advisible to save SVG files as "Plain SVG". For icons it's not a big issue, but in certain cases these Inkscape SVG may not be parsed correctly by every software.